toss/fitness/fitness_function_utils.py

```diff
@@ -52,17 +51,12 @@ def _compute_squared_distance(positions: np.ndarray, constant: float) -> np.ndar
     return np.sum(np.power(positions,2), axis=0) - constant**2
 
 
-def compute_space_coverage(number_of_spacecrafts: int, spin_axis: np.ndarray, spin_velocity: float, positions: np.ndarray, velocities: np.ndarray, timesteps: np.ndarray, radius_min: float, radius_max: float, max_velocity_scaling_factor: float) -> float:
+def compute_space_coverage(number_of_spacecrafts: int, spin_axis: np.ndarray, spin_velocity: float, positions: np.ndarray, velocities: np.ndarray, timesteps: np.ndarray, radius_min: float, radius_max: float, r: np.ndarray, theta: np.ndarray, phi: np.ndarray, bool_tensor: np.ndarray) -> float:
```

Comment:
maybe would make sense to have a simple test for this? Just give it a bool_tensor where e.g. the left hemisphere is all true, and then you give it first only position in the right to see if score > 0, then only in the left to check score == 0 ?

Reply:
I have now added the test (and made some updates in the computation of unique visits).